codalab/lib/bundle_cli.py

```diff
@@ -732,6 +732,7 @@ def do_command(self, argv, stdout=None, stderr=None):
             structured_result = command_fn()
         else:
             try:
+                """
```

Comment:
Why comment this out?

Reply:
Should we just remove it? Seems like this is the wrong way to have an optional profiler anyways.